frontend/src/scenes/comments/Comment.tsx

```diff
@@ -49,6 +51,18 @@ const Comment = ({ comment }: { comment: CommentType }): JSX.Element => {
 
     const isHighlighted = replyingCommentId === comment.id || editingComment?.id === comment.id
     const reactions = emojiReactionsByComment[comment.id] || {}
+    const recordingLinkInfo = getRecordingLinkInfo(comment)
+
+    const handleViewInRecording = (): void => {
+        if (!recordingLinkInfo) {
+            return
+        }
+        if (isViewingRecording(recordingLinkInfo.recordingId) && recordingLinkInfo.unixTimestampMillis) {
+            router.actions.push(recordingLinkInfo.url)
+        } else {
+            window.location.href = recordingLinkInfo.url
+        }
```

Comment:
if we're already watching the recording then we can update the URL so that we jump to timestamp, otherwise navigate to it